apisix/plugins/prometheus/exporter.lua

```diff
@@ -352,13 +360,50 @@ local function etcd_modify_index()
 end
 
 
+local function shared_dict_status()
+    local attr = plugin.plugin_attr(plugin_name)
```

Comment:
We can list the shdict in https://github.com/apache/apisix/blob/master/apisix/cli/ngx_tpl.lua, instead of requiring the users to configure them? Since the usage of shdict is an implementation detail.

Reply:
Something like:
```
{
    "prometheus-metrics",
    "internal-status",
     ...
}
```

To make it simple, we can only provide the must-have shdicts in this first version.

Reply:
I change my mind. We can iterate `ngx.shared` to get all the shdict.

Reply:
> We can list the shdict in https://github.com/apache/apisix/blob/master/apisix/cli/ngx_tpl.lua, instead of requiring the users to configure them? Since the usage of shdict is an implementation detail.

This is what I was trying to do at first, but I don't know how to get all the shdicts.

Reply:
> I change my mind. We can iterate `ngx.shared` to get all the shdict.

I modified to get all ngx.shared.DICT, I found it not a hard job.